src/app/form/(formSteps)/personal-details/3/page.tsx

```diff
@@ -95,7 +95,7 @@ const PersonalDetailsStep3 = () => {
                 </span>
               )}
             </div>
-            <div className="form-first-question-explainer">
+            <div className="form-explainer form-first-question-explainer">
```

Comment:
[dust] Seeing that you changed the CSS, is "form-first-question-explainer" still being used?